policy/modules/contrib/wireguard.te

```diff
@@ -41,6 +42,7 @@ optional_policy(`
 ')
 
 optional_policy(`
+	dbus_send_system_bus(wireguard_t)
```

Comment:
According to `macro-expander`, `dbus_system_bus_client(wireguard_t)` (below) already contains `allow wireguard_t { system_dbusd_t self }:dbus send_msg;`, which is a superset of `dbus_send_system_bus(wireguard_t)`.

Reply:
@milosmalik